build-support/setup_canary_app.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+brew install gnu-sed
+
+# clean
+rm -rf amplified_todo
+
+# generate
+flutter create amplified_todo
+cd amplified_todo
+flutter pub add amplify_flutter && flutter pub add amplify_datastore && flutter pub add amplify_storage_s3 && flutter pub add amplify_analytics_pinpoint && flutter pub add amplify_auth_cognito && flutter pub add amplify_api
```

Comment:
This would install stable versions right now.

```suggestion
# Remove at GA
VERSION=^1.0.0-0
flutter pub add amplify_flutter:$VERSION && \
  flutter pub add amplify_datastore:$VERSION && \
  flutter pub add amplify_storage_s3:$VERSION && \
  flutter pub add amplify_analytics_pinpoint:$VERSION && \
  flutter pub add amplify_auth_cognito:$VERSION && \
  flutter pub add amplify_api:$VERSION
```